src/arrayMethodSort.js

```diff
@@ -4,8 +4,22 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (item1, item2) => {
+    if (String(item1) > String(item2)) {
+      return 1;
+    }
+
+    return -1;
+  }) {
+    for (let j = 0; j < this.length - 1; j++) {
```

Comment:
it's possible more than 2 iterations are needed to complete the sorting
hint: actually, we don't know how many iterations we have to do, but at least one